src/lib/phdi-building-blocks/phdi_building_blocks/fhir.py

```diff
@@ -295,3 +296,38 @@ def _download_export_blob(blob_url: str, encoding: str = "utf-8") -> TextIO:
     text_buffer = io.TextIOWrapper(buffer=bytes_buffer, encoding=encoding, newline="\n")
     text_buffer.seek(0)
     return text_buffer
+
+
+def fhir_server_get(url: str, access_token: str) -> requests.models.Response:
+    """
+    Submit a GET request to a FHIR server given a url and access token for
+    authentication.
+
+    :param url: URL specifying a get request on a FHIR server.
+    :param access_token: A bearer token to authenticate with the FHIR server.
+    """
+
+    header = {"Authorization": f"Bearer {access_token}"}
+    response = requests.get(url=url, headers=header)
+    log_fhir_server_error(response.status_code)
+
+    return response
+
+
+def log_fhir_server_error(status_code: int) -> None:
+    """Given an HTTP status code from a FHIR server's response, log the specified error.
+
+    :param status_code: Status code returned by a FHIR server
```

Comment:
The only other error I can remember seeing coming back from a FHIR server is 403 - forbidden.  This happens when authentication is successful, but the user (or token) isn't authorized for the request they are trying to make.  I still think it would be better to log unexpected status code here, just by adding an else clause:
```
else:
  logging.error(f"FHIR SERVER ERROR - Status code {status_code}")
```

Reply:
How do you feel about:
`    elif str(status_code).startswith(("4", "5")):
        logging.error(f"FHIR SERVER ERROR - Status code {status_code}")` 
        
This way if a non-error status code gets passed the function does nothing.

Reply:
Addressed in [bd4eb97](https://github.com/CDCgov/prime-public-health-data-infrastructure/pull/129/commits/bd4eb97b2ef94e130456a1599c488a95a6821454) along with improvements to the quality of the unit test for `log_fhir_server_error`.